docs/3.x/tech/apis/kids-chat.adoc

```diff
@@ -8,49 +8,46 @@ description: Whitelist APIs that allow your application to access the whitelist
 
 [NOTE.info]
 --
-The whitelist is known as the kids chat on the frontend. In this document it will still be referred to as the whitelist.
+The Whitelist is known as the kids chat on the frontend. In this document it will still be referred to as the Whitelist.
```

Comment:
`kids chat` is referred to as `Kids Chat` in the front end -or more specifically the CleanSpeak Management Interface.